src/riot_sbom/cli.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright (C) 2025 ML!PA Consulting GmbH
+
+SPDX-License-Identifier: MIT
+
+Authors:
+    Daniel Lockau <daniel.lockau@ml-pa.com>
+"""
+
+from .api import run_build
+from .api import load_app_info
+from .api import save_app_info
+from .api import run_plugin
+
+from pathlib import Path
+import logging
+logger = logging.getLogger(__name__)
+from typing import List
+import sys
+
+def main() -> int:
+    from .processing import plugin_registry
+    import argparse
+    parser = argparse.ArgumentParser(description='RIOT SBOM generator')
+    parser.add_argument('--app-dir', type=Path,
+                        help='Path to the directory of the application for which the SBOM should be built. `make -j` should build the application in that directory. Ignored if loading application information from a file, required otherwise.',
```

Comment:
```suggestion
                        help='Path to the directory of the application for which the ' \
                             'SBOM should be built. `make -j` should build the ' \
                             'application in that directory. Ignored if loading ' \
                             'application information from a file, required otherwise.',
```

I think this is how you can break up long strings in Python. Please also adapt for the following lines.